portal/templates/landing.html

```diff
@@ -17,13 +27,18 @@
         <div id="wellContainer">
             <div class="tnth-splash">
                 <div id="wellContent">
-                  <div class="headline">
+                    <div class="headline">
                         <div style="margin-bottom: 1em">{{ app_text('landing title') }}</div>
                         {{ app_text('landing sub-title') }}
                     </div>
                     <a href="/go" class="btn feature-btn feature-btn-primary">{{ _("Register") }}</a> &nbsp;&nbsp; <a href="/user/sign-in" class="btn feature-btn feature-btn-primary">{{ _("Login") }}</a>
-
+                    {% if timed_out %}
+                        <div id="timeOutMessageContainer">
+                            You have been logged out due to inactivity. Please log in again to continue.
```

Comment:
We should wrap this in a get text call for i18n, namely:
`{{ _("You have been logged out due to inactivity. Please log in again to continue.") }}`